### PR TITLE
Backup: Search by name and id

### PR DESCRIPTION
**Pull request by rsalevsky**
This makes it possible to download an image by the name. Specilly for
the crowbar-client this is very helpful.

### CODE REVIEW

**Comment by tboerger** [comment]
:+1: as this is a snippet i gave you from another project :D

File: crowbar_framework/app/models/backup.rb [hunk 1]
```diff
@@ -168,4 +168,20 @@ def validate_upload_file_extension
 
     errors.add(:base, I18n.t("backups.validation.invalid_file_extension"))
   end
+
+  class << self
+    def find_using_name(name)
```
Comment (MaximilianMeister) [inline crowbar_framework/app/models/backup.rb]: so this method can be used for finding records via name **or** id?

Comment (MaximilianMeister) [inline crowbar_framework/app/models/backup.rb]: can we name it `find_using_name_or_id(name_or_id)`?

**Comment by MaximilianMeister** [comment]
+1

**Pull request merged**